src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/indexed_fields_table/components/table/table.js

```diff
@@ -39,7 +39,16 @@ export class TableComponent extends PureComponent {
     return value ? <EuiIcon type="dot" color="secondary" aria-label={label}/> : <span/>;
   }
 
-  renderFieldName(name, isTimeField, label, content) {
+  renderFieldName(name, isTimeField, intl) {
```

Comment:
Minor nit but we don't need to pass `intl`, we can just destructure it from props locally:

```jsx
const { intl } = this.props;
```